src/mame/atari/cops.cpp

```diff
@@ -56,15 +57,20 @@ namespace {
 #define AUX_REGISTER 1
 
 #define MAIN_CLOCK XTAL(4'000'000)
+#define DACIA_CLOCK XTAL(3'686'400)
```

Comment:
You can use 3.6864_MHz_XTAL for this and not have to have a #define since it's a common value with ACIAs.

Reply:
There's a few crystals I haven't done that with, there's even a small one on the RTC I missed so I'll deal with those.

As for the 6552 - there's just enough different that it is an issue to merge with 6551 (the addition of comparison registers, the reworking of some of the status pins to do different things. It's effectively a double up of a slightly different design). However, I have at least put this in a more 'modern' framework which might facilitate a merge if others feel it suitable.

The most recent commit addresses the crystals, uses the more modern shorthand and structure for the devices (not the driver as that's preexisting code) and adds the 'Ltd' to fit the company trading name. Hope this addresses the concerns and that, if nothing else, the 1450 might be useful for something people might like ;).

Reply:
Beyond the concerns that were already brought up, I believe this PR should go in without any fuss regarding the 6552 vs. 6551 distinction.

Given the current implementation of the 6551 device using the read/write paradigm, it'd be difficult to override any specific register behavior for the sake of the 6552 device.

I'd much rather see these changes go in to improve the overall driver, and then have a separate pull request for improving the 6551/6552 split (since it seems like @james-wallace-ghub is planning to do that), than to stand on ceremony over a tangential peripheral chip.

Reply:
Yes, I think my next plan would be to try to unify based on the 'new' structure rather than the old, but I want to try to get my head around that first. In this situation, the drivers using the 6551 are unaffected, and then I can merge and test at relative leisure.

Reply:
Yeah, I'm fine with the 6552 as it was originally submitted, I just wanted the crystal constant used and the commented-out logerrors resolved.  Please resist the temptation to pile on more work when I'm just asking for a few simple changes, it makes things easier for me and for you.

Reply:
If leaving it at this is ok, I'm not intending to go further. I just thought a map with the 6552 would help for any future work outside of this scope.